TDP/tdp-44.md

```diff
@@ -0,0 +1,69 @@
+
+Since its inception, the mStableDAO has gone through several iterations in an attempt to improve its efficiency. Examples include [improvements to the ProtocolDAO structure](https://forum.mstable.org/t/mstable-protocoldao-migration-and-upgrade/352), the [creation of the Asset Management subDAO](https://forum.mstable.org/t/proposal-creation-initial-funding-of-the-mstabledao-asset-management-subdao/568) and [creation of the Funding subDAO](https://forum.mstable.org/t/future-funding-of-mstable/543).
+
+Fundamentally, the ProtocolDAO and TreasuryDAO have proven to serve their purpose in executing the will of the Meta Governors. However, we have learned that slow execution in multisigs has not always been resolved by reducing the size of the signer group and that in fact, 2/3 multisigs may be more susceptible to delays than larger structures such as 4/7 multisigs. This is evidenced by the efficiency of the current ProtocolDAO, and the delays that have been experienced in smaller signer structures within the Asset Management and Funding subDAOs. Instead, delays tend to be caused by a lack of clarity in the responsibilities of signers and a lack of availability and engagement, as well as time zone differences in some cases.
+
+**Inefficiencies that have been identified in the current DAO structure include:**
+
+- The complexity of the current DAO structure and a lack of clarity around the responsibilities and purpose of each entity
+- A number of disengaged signers on the TreasuryDAO, leading to long delays in transaction execution
+- Lack of thorough review and verification of transactions due to low involvement in the governance or decision-making process
+- Compromised security in the Asset Management subDAO, which now custodies around $3.5m in assets, without significant gains in efficiency
+- Growth of off-chain entity operations and associated costs, leading to more centralization and complexity
+- A disconnect between the way the core team works together and the structure of the DAO
+- A lack of clear purpose and deliverables for entities funded by the mStableDAO
+
+This proposal aims to address the inefficiencies outlined above through simplification of DAO structure and clarification of the purpose, responsibilities, and decision-making processes within each entity.
+
+Aside from aligning with a new vision and focus for mStable, the timing of this review also coincides with a number of key dates that would have already required individual governance processes, so it’s the perfect time to consolidate all of these requests in this restructuring proposal.
+
+## Specification
+
+1.) Begin parallel rollout of TDP 44 for the election process, TDP 44 for the GrantsDAO and Community subDAO consolidation & review of the GrantsDAOs success after it’s first year of operation, as well as TDP 44 for the proposed compensation model under which the mStableDAO will fund its workforce.

```

Comment:
TDP 45 for the election process, TDP 46 GrantsDAO and Community subDAO consolidation & review of the GrantsDAOs success after it’s first year of operation, as well as TDP 47 for the proposed compensation model under which the mStableDAO will fund its workforce.

Reply:
add links later to the proposals once they are up :)

Reply:
I thought we agreed to name them all TDP 44?